WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -116,13 +116,35 @@ private extension PaymentCaptureOrchestrator {
             return nil
         }
 
+        var customerName: String?
+
+        if let firstName = order.billingAddress?.firstName {
+            customerName = firstName
+        }
+
+        if let lastName = order.billingAddress?.lastName {
+            if customerName == nil {
+                customerName = lastName
+            } else {
+                customerName? += " " + lastName
+            }
+        }
```

Comment:
This might be OK considering that we're launching only in the US first, but I wonder if we need to [localize full names](https://www.w3.org/International/questions/qa-personal-names) better. Luckily, it seems Apple provides a [PersonNameComponentsFormatter](https://developer.apple.com/documentation/foundation/personnamecomponentsformatter) for that.

Reply:
I agree! This is (presently) how the backend does it as well. I'll open an issue here and there

Reply:
https://github.com/woocommerce/woocommerce-ios/issues/4217